src/qt/sendcoinsentry.cpp

```diff
@@ -97,7 +97,9 @@ void SendCoinsEntry::clear()
     ui->payTo->clear();
     ui->addAsLabel->clear();
     ui->payAmount->clear();
-    ui->checkboxSubtractFeeFromAmount->setCheckState(Qt::Unchecked);
+    if(model && model->getOptionsModel()){
```

Comment:
```suggestion
    if (model && model->getOptionsModel()) {
```

There is a very useful tool to check commit code style before pushing -- [`clang-format-diff.py`](https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy).